app/views/layouts/_header.html.erb

```diff
@@ -21,18 +21,15 @@
 </ul>
 
 <ul id="Donate" class="dropdown-content">
-  <li><%= link_to 'Square Cash', square_path %></li>
-  <li role="separator" class="divider"></li>
-  <li><%= link_to 'Dwolla', dwolla_path %></li>
-  <li role="separator" class="divider"></li>
-  <li><%= link_to 'Online', donate_path %></li>
-  <li role="separator" class="divider"></li>
-  <li><%= link_to 'Swag', cottonbureau_path %></li>
-  <li role="separator" class="divider"></li>
-   <li><%= link_to 'AmazonSmile', amazon_path %></li>
+  <li><a class="dbox-donation-button" href="https://donorbox.org/operationcode" 
```

Comment:
Can we get this link to be top level in the menu? It would end up looking like the Blog link?

Could you also look at removing the icon next to the 'Donate' link?